aten/src/ATen/native/native_functions.yaml

```diff
@@ -1419,7 +1419,7 @@
   dispatch:
     CUDA: batch_norm_elemt_cuda
 
-- func: batch_norm_gather_stats(Tensor input, Tensor mean, Tensor invstd, Tensor? running_mean, Tensor? running_var, float momentum, float eps, int count) -> (Tensor, Tensor)
+- func: batch_norm_gather_stats(Tensor input, Tensor mean, Tensor invstd, Tensor? running_mean, Tensor? running_var, float momentum, float eps, int[] counts) -> (Tensor, Tensor)
```

Comment:
I believe this is still BC-breaking. As mentioned above, can you try keep both the original one and the new one?

Reply:
@mrshenli you should probably also deprecate the old one so we can get rid of it in the future.